releases/unreleased/upgrade-to-grimoirelab-192.yml

```
@@ -0,0 +1,8 @@
+---
+title: Upgrade to GrimoireLab 1.6.0
```

This title is wrong.
```suggestion
title: Upgrade to GrimoireLab 1.9.2
```